libbeat/docs/shared-autodiscover.asciidoc

```diff
@@ -185,6 +196,8 @@ The `kubernetes` autodiscover provider has the following configuration settings:
 `namespace`:: (Optional) Select the namespace from which to collect the
   metadata. If it is not set, the processor collects metadata from all
   namespaces. It is unset by default.
+`cleanup_timeout`:: (Optional) Time of inactivity to consider we can stop the
```

Comment:
All butter points here are written in an imperative form - _use_, _specify_, _select_. Do you think you can rephrase the description for `cleanup_timeout` to sound similarly and be straightforward?
Also, I think we shouldn't use the subject here (we, you, etc..). I think I saw `you` somewhere in this doc.

Reply:
```suggestion
`cleanup_timeout`:: (Optional) Specify the time of inactivity before stopping the
```